scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -66,8 +66,10 @@
           {% set ns.allele0 = variant.chromosome + variant.position|string %}
           {% if variant.dismiss_variant %}
               <tr class="dismiss">
+          {% elif variant._id in case.causatives %}
+             <tr class="causative">
```

Comment:
Ah, ouch! My bad! The STRs. They are sort of greeninsh when normal, so green will perhaps not be so great for causative, or what do you think? Unless green in some other way, like with outline or font color changed? Suggestions?

Reply:
I changed the normal STRs color to primary blue. Since green is the color of solved cases in the UI I think it is consistent to use it for causative variants as well. Less consistent to have the same green in solved cases and normal STRs. The blue is a bit dark for my taste but I think it works. What do you think?
<img width="850" alt="Screen Shot 2020-08-18 at 15 35 31" src="https://user-images.githubusercontent.com/6919697/90519573-81dfd600-e168-11ea-85b7-41514371cd4d.png">

Reply:
Thats excellent! Agreed, a lighter blue would have been more appealing here, but recurring color is also important.